include/tvm/relay/logging.h

```diff
@@ -27,23 +27,73 @@
 #define TVM_RELAY_LOGGING_H_
 
 #include <dmlc/logging.h>
-#include <string>
+#include <tvm/logging.h>
 #include <cstdlib>
 #include <iostream>
+#include <string>
+#include <type_traits>
 
 namespace tvm {
 namespace relay {
 
-static bool logging_enabled() {
-  if (auto var = std::getenv("RELAY_LOG")) {
+#ifdef USE_RELAY_DEBUG
+struct EnableRelayDebug {
+  static constexpr bool value = true;
+};
+
+static inline bool logging_enabled() {
+  if (auto var = std::getenv("USE_RELAY_LOG")) {
```

Comment:
I worried a bit about this duplicated logic. There are now three places such logging info def existed.

Consider directly use ```DLOG(INFO)``` which offers similar features https://github.com/dmlc/dmlc-core/blob/master/include/dmlc/logging.h#L181

Reply:
As @jroesch suggested, we had this to make logging quiet/verbose without recompilation but just through an env var. We added the RELAY_LOG for logging just in relay namespace.